Source/WebCore/rendering/ImageQualityController.cpp

```diff
@@ -113,7 +113,7 @@ std::optional<InterpolationQuality> ImageQualityController::interpolationQuality
     return std::nullopt;
 }
 
-InterpolationQuality ImageQualityController::chooseInterpolationQuality(GraphicsContext& context, RenderBoxModelObject* object, Image& image, const void* layer, const LayoutSize& size)
+InterpolationQuality ImageQualityController::chooseInterpolationQuality(GraphicsContext& context, RenderBoxModelObject* object, Image& image, const void* layer, const LayoutSize& layoutSize)
```

Comment:
I think using destinationSize is more descriptive of the nature of the rectangle. Also I think FloatSize can be used instead of LayoutSize.

Reply:
You mean `FloatSize& destinationSize` instead of `LayoutSize& layoutSize`?

Reply:
yes.

Reply:
Done